core/src/main/java/com/auth0/authentication/AuthenticationAPIClient.java

```diff
@@ -447,7 +447,7 @@ public ChangePasswordRequest changePassword(String email) {
                 .addParameters(parameters);
     }
 
-    protected ParameterizableRequest<Token> loginWithResourceOwner() {
+    public ParameterizableRequest<Token> loginWithResourceOwner() {
```

Comment:
I can think of some uses since the parameters can change a lot if you use DB connection vs SMS connection. If we keep it just add `javadoc`